samples/sample_vpp/src/sample_vpp_parser.cpp

```diff
@@ -414,6 +414,16 @@ mfxU32 Str2FourCC( msdk_char* strInput )
     {
         fourcc = MFX_FOURCC_UYVY;
     }
+#if (MFX_VERSION >= 1027)
+    else if ( 0 == msdk_stricmp(strInput, MSDK_STRING("y210")) )
+    {
+        fourcc = MFX_FOURCC_Y210;
+    }
+    else if ( 0 == msdk_stricmp(strInput, MSDK_STRING("y410")) )
+    {
+        fourcc = MFX_FOURCC_Y410;
+    }
+#endif
```

Comment:
could you add else block and throw an exception if we can't find a corresponding if block

Reply:
@dmitryermilov, I did this via call vppPrintHelp() and return MFX_ERR_UNSUPPORTED status.